src/platforms/web/util/element.js

```diff
@@ -19,15 +19,17 @@ export const isHTMLTag = makeMap(
   'button,datalist,fieldset,form,input,label,legend,meter,optgroup,option,' +
   'output,progress,select,textarea,' +
   'details,dialog,menu,menuitem,summary,' +
-  'content,element,shadow,template'
+  'content,element,shadow,template,' +
+  'big,blockquote,iframe,keygen,tfoot'
 )
 
 // this map is intentionally selective, only covering SVG elements that may
 // contain child elements.
 export const isSVG = makeMap(
   'svg,animate,circle,clippath,cursor,defs,desc,ellipse,filter,font-face,' +
   'foreignObject,g,glyph,image,line,marker,mask,missing-glyph,path,pattern,' +
-  'polygon,polyline,rect,switch,symbol,text,textpath,tspan,use,view',
+  'polygon,polyline,rect,switch,symbol,text,textpath,tspan,use,view,' +
+  'clipPath,linearGradient,radialGradient,stop',
```

Comment:
As commented above, "this map is intentionally selective, only covering SVG elements that may contain child elements." - this also only applies to tags that may appear as component/template root nodes, so these additions are not needed.